oauth/oauth.dart

```diff
@@ -0,0 +1,57 @@
+import 'package:flutter_appauth/flutter_appauth.dart';
+
+class AuthRepository {
+  AuthRepository();
+
+  final _auth = const FlutterAppAuth();
+
+  @override
+  Future<Authorization?> authorize() async {
+    try {
+      final response = _auth.authorizeAndExchangeCode(
```

Comment:
Does this open web view on it's own?

Reply:
Yes, it automatically opens the web-view.